docs/pages/button.js

```diff
@@ -27,88 +32,75 @@ card(
         required: false,
         defaultValue: null,
         description: [
-          'Supply a short, descriptive label for screen-readers to replace button texts that do not provide sufficient context about the button component behavior. Texts like `Click Here,` `Follow,` or `Read More` can be confusing when a screen reader reads them out of context. In those cases, we must pass an alternative text to replace the button text.',
-          'Use IconButton if you only need a button with an icon and no text.',
-          'Accessibility: It populates aria-label. Screen readers read the `accessibilityLabel` prop, if present, instead of `text`.',
+          'Label to provide more context around Button’s function or purpose. See the [Accessibility guidelines](/accessibility) to learn more.',
         ],
-        href: 'accessibilityLabel',
+        href: 'Accessibility-props:-controls-expanded-and-popup',
       },
       {
         name: 'accessibilityControls',
         type: 'string',
         required: false,
         defaultValue: null,
         description: [
-          'Specify the `id` of an associated element (or elements) whose contents or visibility are controlled by a button component so that screen reader users can identify the relationship between elements.',
-          'Optional with button-role + button-type buttons.',
-          'Accessibility: It populates aria-controls.',
+          'A unique id indicating the element or elements whose contents or visibility are controlled by Button. See the [Accessibility guidelines](/accessibility) to learn more.',
         ],
-        href: 'accessibility',
+        href: 'Accessibility-props:-controls-expanded-and-popup',
       },
       {
         name: 'accessibilityExpanded',
         type: 'boolean',
         required: false,
         defaultValue: null,
         description: [
-          'Indicate that a button component hides or exposes collapsible components and expose whether they are currently expanded or collapsed.',
-          'Optional with button-role + button-type buttons.',
-          'Accessibility: It populates aria-expanded.',
+          'Needed if Button controls a grouping of other elements, e.g. Dropdown or Flyout. This is used to indicate if the controlled grouping is currently expanded or collapsed. See the [Accessibility guidelines](/accessibility) to learn more.',
```

Comment:
I think `Button controls a grouping` can be simplified to 
"Button controls the visibility of other elements, e.g. Dropdown or Flyout"

I think `grouping` is unrelated

Reply:
Good call. Fixed!